spec/features/platform_admin_can_see_products_by_category_spec.rb

```diff
@@ -0,0 +1,88 @@
+require "rails_helper"
+
+feature "anyone can view auctions by category" do
```

Comment:
The name of the spec file is different than the feature test. Am I missing something here? hehe